Tests/FluentAssertions.Json.Specs/FluentAssertions.Json.Specs.csproj

```diff
@@ -9,10 +9,10 @@
 
     <ItemGroup>
         <PackageReference Include="AutoFixture" Version="4.17.0" />
-        <PackageReference Include="FluentAssertions" Version="5.10.3" />
+        <PackageReference Include="FluentAssertions" Version="6.0.0" />
```

Comment:
Seems like a leftover from packages.config days as it is included through FA.json.